heartbeat/monitors/monitor.go

```diff
@@ -256,3 +321,16 @@ func (m *Monitor) Stop() {
 
 	m.stats.stopMonitor(int64(m.endpoints))
 }
+
+// WithMonitorMeta ensures that monitor.{id,name,type} are present.
+func WithMonitorMeta(id string, name string, typ string, origJob Job) Job {
+	return WithFields(
+		common.MapStr{
+			"monitor": common.MapStr{
+				"id":   id,
+				"name": name,
+				"type": typ},
```

Comment:
Nit: `}` should probably be on a new line.

Reply:
Good find , weird go fmt didn't catch that.